crates/sys/src/lib.rs

```diff
@@ -5,14 +5,7 @@
 
 // Features that identify thumbv7em-none-eabihf.
 #[cfg(not(any(
-    all(
-        target_arch = "arm",
-        target_feature = "thumb2",
-        target_feature = "v7",
-        target_feature = "dsp",
-        target_os = "none",
-        target_abi = "eabihf",
-    ),
```

Comment:
I would prefer that we comment out the unstable components, like we were doing with `target_abi` before:
```suggestion
// Features that identify thumbv7em-none-eabihf.
// Until target_feature is stable, this also permits TODO.
#[cfg(not(any(
    all(
        target_arch = "arm",
        //target_feature = "thumb2",
        //target_feature = "v7",
        //target_feature = "dsp",
        target_os = "none",
        target_abi = "eabihf",
    ),
```

Reply:
Sure. Added this back in.